index.js

```diff
@@ -134,6 +141,16 @@ const ContainerService = {
 
   // utility methods
 
+  _totalContainerTabs() {
+    let totalContainerTabs = 0;
+    for (const userContextId in this._identitiesState) {
+      if (this._identitiesState.hasOwnProperty(userContextId)) {
```

Comment:
remove this line. Just do:

for (const userContextId in this._identitiesState) {
  totalContainerTabs += this._identitiesState[userContextId].openTabs;
}